Mathlib/Combinatorics/SimpleGraph/Path.lean

```diff
@@ -645,6 +645,41 @@ theorem edges_toPath_subset {u v : V} (p : G.Walk u v) : (p.toPath : G.Walk u v)
 
 end Walk
 
+/-! ### Circuits to cycles -/
+
+-- The type of cycles for a closed walk.
+abbrev Cycle (u : V) := { p : G.Walk u u // p.IsCycle }
+
+namespace Walk
+
+variable {G} [DecidableEq V]
+
+/-- Given a closed walk, produces a closed walk removing subwalks between repeated vertices
+through the use of the bypass definition.
+If the walk provided is a circuit, this results in a cycle, as
+seen in `SimpleGraph.Walk.isCycle_shortcircuit`
+This idea is packaged up in `SimpleGraph.Walk.ToCycle`. -/
+def shortcircuit {u : V} : G.Walk u u → G.Walk u u
```

Comment:
```suggestion
def shortCircuit {u : V} : G.Walk u u → G.Walk u u
```

Reply:
This is closer to the usual naming style for mathlib! Note you'll need to adjust in the next lemma too (and that one should be `isCycle_shortCircuit`